DeAceroDLabs/marketplace-frontend
# [DMP-730] Mostrar cantidad de productos en carrito

## What is done

[[DMP-730] [Frontend] Mostrar cantidad de productos en carrito](https://deacero.atlassian.net/browse/DMP-730)

## How is done

1. Added HTML to display quantity of products
2. Updated Snapshots

## How to test

1. Run the project via `npm start`
3. Go to ShoppingCart with added products
4. If product quantity varies, the number should display it correctly

## Evidence

===================================BEFORE==========================================
![image](https://user-images.githubusercontent.com/116199447/207261614-718aa35e-9f82-4cb0-8e9f-36f848217bd6.png)


===================================AFTER===========================================
![image](https://user-images.githubusercontent.com/116199447/207261358-b6f63df7-87af-480a-adfe-030b73548971.png)
